tests/integration/tests/test_sanity_checks.py

```diff
@@ -9,7 +9,7 @@
 import pytest
 from commonlib.utils import get_findings
 
-CONFIG_TIMEOUT = 60
+CONFIG_TIMEOUT = 300
```

Comment:
do we still need to change the timeout? (as we excluded `file` type)

Reply:
In the latest run, all results were retrieved within 2 minutes. I will update the timeout to 120 seconds. However, it's advisable to keep some buffer time due to potential cloud factors that may affect the result. Nevertheless, if the result is retrieved before reaching the timeout, the test run will be considered finished and won't wait.